.github/workflows/tests.yml

```
@@ -34,14 +34,18 @@ jobs:
         run: yarn lint
 
   test:
-    name: Run tests
+    name: Run tests on ${{ matrix.os }} and node ${{ matrix.node_version }}
```

To get a quicker node version display in github action I suggest the following:

```suggestion
    name: Run tests with node ${{ matrix.node_version }} on ${{ matrix.os }}
```